<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transfer_Funds</class>
 <widget class="QWidget" name="Transfer_Funds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>550</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Main Page</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Process_LE">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>931</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Transfer Funds</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>1171</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 900 10pt &quot;Segoe UI&quot;;
font: 900 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Enter Account Number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Account_Number_LE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 900 10pt &quot;Segoe UI&quot;;
font: 900 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Enter Funds Amount:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Funds_Amount_LE"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>1101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 16pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;Transfer Funds to Another Account&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
